Pieces/Piece.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Chess.Board;
+namespace Chess.Pieces
+{
+    abstract class Piece
+    {   
+        protected bool first_move = true;
+        public string name;
+        protected string piece_index;
+        protected ConsoleColor team_color;
```

Comment:
G26. sugeruję uzywania własnych typów Enum, typ ConsoleColor jest mylący, skoro zmienna `team_color` ma reprezentować białe lub czarne pionki, a nie kolor na konsoli
```suggestion
        protected TeamColor Color;
```

Reply:
Poprawiono